pyproject.toml

```diff
@@ -1,55 +1,49 @@
 [build-system]
-requires = ["setuptools>=61.0"]
-build-backend = "setuptools.build_meta"
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
 
-[project]
+[tool.poetry]
 name = "ubuntu_package_download"
-version = "0.0.6"
+version = "0.0.7"
 description = "Helpful utility to download Ubuntu debian packages"
-readme = "README.rst"
+license = "GPL-3.0-only"
 authors = [
-  {name = "Philip Roche", email = "phil.roche@canonical.com"}
+  "Philip Roche <phil.roche@canonical.com>",
 ]
 maintainers = [
-  {name = "Philip Roche", email = "phil.roche@canonical.com"}
-]
-classifiers = [
-
-]
-license = {text = "GNU General Public License v3"}
-dependencies = [
-    "typer",
-    "launchpadlib",
-    "python-debian",
-    "rich",
-    "typing-extensions",
+  "Philip Roche <phil.roche@canonical.com>",
```

Comment:
You should add yourself at least to the maintainers list if not as an author

Reply:
@catmsred Added myself to the maintainer list. :)